src/Psl/Iter/enumerate.php

```diff
@@ -15,7 +15,7 @@
  *
  * @param iterable<Tk, Tv> $iterable
  *
- * @return Iterator<int, array{0: Tk, 1: Tv}>
```

Comment:
what is the difference between `array` and `strict-array`?

Reply:
See https://psalm.dev/docs/annotating_code/type_syntax/array_types/#sealed-object-like-arrays, basically `strict-array`s cannot have extra keys, thus you can't pass for example an `array{a: int, b: int, c: \Exception}` into an `avg` function that takes an `array{a: int, b: int}`.

Reply:
ah! okay :) makes sense.